lib/mutant/integration/rspec.rb

```diff
@@ -33,7 +33,8 @@ class Rspec < self
       #
       # @api private
       #
-      def initialize
+      def initialize(*args)
```

Comment:
Is it necessary to use `*args` here? Can you just use `*`, and then use `super` without parenthesis?